engine/api/workflow/dao.go

```diff
@@ -918,7 +918,22 @@ func Update(ctx context.Context, db gorp.SqlExecutor, store cache.Store, proj sd
 
 	// Delete workflow data
 	if uptOption.OldWorkflow != nil && uptOption.OldWorkflow.ID != 0 {
```

Comment:
OldWorkflow can't be nil, must not be a pointer

Reply:
done